src/providers/push-notifications/push-notifications.ts

```diff
@@ -1,7 +1,7 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { FCM } from '@ionic-native/fcm';
-import { App, NavControllerBase } from 'ionic-angular';
```

Comment:
Although not visible in this diff, it's a good idea to also remove the imported pages in this file.

Reply:
done! thanks